numba/tests/test_parfors.py

```diff
@@ -1622,6 +1622,18 @@ def test_impl(a):
         x = np.arange(10)
         self.check(test_impl, x)
 
+    @skip_parfors_unsupported
+    def test_inplace_binop(self):
+        def test_impl(a, b):
+            b += a
+            return b
+
+        X = np.ones(10)
+        Y = np.zeros(10)
```

Comment:
The test cannot detect if the transformed code is producing the correct result because the addition is adding zeros. 

perhaps change to arrays with non-zeros
```suggestion
        X = np.arange(10) + 10
        Y = np.arange(10) + 100
```